app/src/main/java/com/example/nvdovin/weatherapp/factory/RetrofitFactory.java

```diff
@@ -26,9 +26,9 @@ public RetrofitFactory() {
                 .build();
     }
 
-    public FirstModel getData(String query_city_name) throws IOException {
+    public Model getData(String query_city_name) throws IOException {
```

Comment:
What does "Model" mean? Also pay attention on how do you name variables see please this link: http://www.oracle.com/technetwork/java/codeconventions-135099.html